<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewVersionWin</class>
 <widget class="QDialog" name="NewVersionWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>781</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Serif</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Creazione nuova versione</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern</family>
     <pointsize>10</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Inserire i parametri necessari per la creazione della nuova versione</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome progetto</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome versione</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="versionName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="major">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sub">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="revision">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="resetNumberBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>azzera</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Eredita</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fromWho">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Versione precedente</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Versione master</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Versione numero</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>113</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Formato nome versione</string>
   </property>
  </widget>
  <widget class="QListWidget" name="flagsList">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nome progetto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Separatore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numero versione maggiore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Separatore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numero sottoversione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Separatore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numero revisione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fine dichiarazione</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="flagChose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Numero versione maggiore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Separatore</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numero sottoversione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Numero revisione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nome versione</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nome progetto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fine dichiarazione</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rimuovi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Svuota</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero elementi: </string>
   </property>
  </widget>
  <widget class="QLabel" name="flagsCount">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Informazioni creazione</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica versione</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>50</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi a</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Progetto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Strumento 1</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewVersionWin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewVersionWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
